src/ir/abstract.h

```diff
@@ -43,16 +43,10 @@ enum Op {
 inline UnaryOp getUnary(Type type, Op op) {
   switch (type) {
     case i32: {
-      switch (op) {
```

Comment:
these are sort of for future use ;) but may as well remove them until that future happens, if it ever does...

Reply:
thought so :) the warning here was `C4065: switch statement contains 'default' but no 'case' labels`